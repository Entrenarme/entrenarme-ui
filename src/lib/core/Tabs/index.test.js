import * as React from 'react';
import { render } from 'react-testing-library';
import 'react-testing-library/cleanup-after-each';

import SwitchTab from './index';

test('renders "Paragraph" with no props', () => {
  const { container } = render(
    <SwitchTab>
      <div>
        <p>Tab 1</p>
        <p>Tab 2</p>
        <p>Tab 3</p>
      </div>
      <div>
        <p>bla1</p>
        <div>
          <p>bla2</p>
        </div>
        <p>bla3</p>
      </div>
    </SwitchTab>,
  );
  expect(container.firstChild).toMatchSnapshot();
});
