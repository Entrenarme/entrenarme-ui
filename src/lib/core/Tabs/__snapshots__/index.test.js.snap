// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders "Paragraph" with no props 1`] = `
<div>
  <p>
    Tab 1
  </p>
  <p>
    Tab 2
  </p>
  <p>
    Tab 3
  </p>
</div>
`;
